src/common/sorted-table/index.js

```diff
@@ -450,11 +516,14 @@ export default class SortedTable extends Component {
     const nVisibleItems = this._getVisibleItems().length
 
     const hasGroupedActions = !isEmpty(groupedActions)
-    const hasIndividualActions = !isEmpty(individualActions)
+    const hasIndividualActions = !isEmpty(props.individualActions)
 
     const nColumns = props.columns.length + (hasIndividualActions ? 2 : 1)
 
-    const paginationInstance = (
+    const displayPagination = paginationContainer === undefined && nVisibleItems < nAllItems
```

Comment:
If `this.state.itemsPerPage === 30`, `nAllItems === 20` and `nVisibleItems === 10`, the pagination will show but it shouldn't, right?